slatedb/src/db/builder.rs

```diff
@@ -264,13 +305,18 @@ impl<P: Into<Path>> DbBuilder<P> {
 
         // Setup the components
         let stat_registry = Arc::new(StatRegistry::new());
-        let sst_format = SsTableFormat {
+        let mut sst_format = SsTableFormat {
             min_filter_keys: self.settings.min_filter_keys,
             filter_bits_per_key: self.settings.filter_bits_per_key,
             compression_codec: self.settings.compression_codec,
             ..SsTableFormat::default()
         };
 
+        // Set block size if configured
```

Comment:
instead of a mut sst_format, can you do:

```rust
        let mut sst_format = SsTableFormat {
            min_filter_keys: self.settings.min_filter_keys,
            filter_bits_per_key: self.settings.filter_bits_per_key,
            compression_codec: self.settings.compression_codec,
            block_size: block_size.unwrap_or_default(),
            ..SsTableFormat::default()
        };
```